pkg/cmd/build/download.go

```diff
@@ -38,7 +38,8 @@ func NewCmdBuildDownload(f *factory.Factory) *cobra.Command {
 				return err
 			}
 			if bld == nil {
-				return fmt.Errorf("could not resolve a build")
+				fmt.Printf("No build found.\n")
+				return nil
```

Comment:
I've just realised this is going to cause problems with our writing to stdout. Because this could be run while a spinner is being shown, outputting anything else at the same time causes problems. Not sure what we are supposed to do about that at the moment.

One option is to look at custom error types. So we can return an error of a certain type, and then the consuming function can look at the error and decide whether to "ignore" it

Reply:
Oh actually, this isn't inside another function inside a spinner so it should be okay.

I think the only benefit would be to use `cmd.OurOrStdout` to write to instead